docs/unraid-os/system-administration/maintain-and-update/changing-the-flash-device.md

```
@@ -0,0 +1,254 @@
+</TabItem>
+<TabItem value="manual" label="Manual copy">
+
+1. Shut down your server and remove the flash device.
+2. Insert it into another computer.
+3. Copy all files from the flash drive to a safe backup location.
+
+</TabItem>
+</Tabs>
+
+<details>
+<summary><strong>What if I can't back up my device?</strong> - Click to expand/collapse</summary>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: strong

(MD033, no-inline-html)